pkg/chart/metadata_test.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package chart
+
+import (
+	"testing"
+)
+
+func TestValidate(t *testing.T) {
+	tests := []struct {
+		md  *Metadata
+		err error
+	}{
+		{
+			nil,
+			ValidationError("chart.metadata is required"),
+		},
+		{
+			&Metadata{Name: "test"},
```

Comment:
in this particular case, the error may crop up as either the version OR the apiVersion field being required, because both fields are missing. Can we add a Version field here as well so that we can explicitly check for the apiVersion error as being done on lines 40 and 41?